wordhelper/anagram.py

```diff
@@ -56,22 +86,64 @@ def find(self):
 				anagrams += words_by_character_frequency[histogram]
 		return anagrams
 
-	def find_multi(self, words_so_far=[]):
+	def find_multi_helper(self, use_words_until, words_so_far):
+		"""
+		Helper function for find_multi
```

Comment:
These comments are great, but I more meant comments in the code that explain the algorithm. For example, below you could add a comment that explains the if condition. It isn't immediately obvious that the reason you have a max word like this is because you're starting from the back of your dictionary and shrinking it as you iterate

Reply:
find() or single word anagram uses a dictionary look up to find the anagrams and no search is involved so that method does not apply to it.